### ISSUE TITLE
Add Support for Database

### ISSUE THREAD

**Issue by foreverXZC** [gruntwork-io/terratest#136]
After I read most of code in modules, I found that there are no particular functions related to database. Therefore, if I would like to deploy a database (i.e. Microsoft SQL Server) using terraform, I have to write my own test code to check whether the database runs correctly.

As a result, I wonder whether it is necessary to add methods to support database operations, such as connection, executing SQL commands, etc. I have written several functions to help my own validation, but I really want to know whether adding such kind of features to terratest is important.

**Comment by brikis98** [comment]
Yes please! We've been meaning to add just such helpers for a few popular DBs (MySQL, Postgres, etc), but have not had time yet. PRs very welcome.
